Resources/Prototypes/_NF/Catalog/Fills/Items/briefcases.yml

```diff
@@ -0,0 +1,85 @@
+- type: entity
+  parent: BaseItem
+  id: SpaceCashExpedition
+  name: syndicate naval operative bundle
+  abstract: true
+  noSpawn: true
+  suffix: Money
+  components:
+  - type: Sprite
+    sprite: Objects/Storage/Briefcases/briefcase_brown.rsi
+    state: icon
+  - type: Item
+    size: Ginormous
+    sprite: Objects/Storage/Briefcases/briefcase_brown.rsi
+  - type: CargoSellBlacklist # We dont want this to be sold by mistake, and so you cannot sell this on a cargo depo
+
+- type: entity
+  parent: SpaceCashExpedition
+  id: SpaceCash5000Expedition
+  name: spesos (5000)
+  noSpawn: true
+  components:
+  - type: SpawnItemsOnUse
```

Comment:
so is this just like a briefcase styled wrapping paper over some spesos? why not ive legit ones filled with stacks of 5k or something? this confuses me just a tad